packages/core/src/components/SingleSelect/SingleSelect.tsx

```diff
@@ -214,6 +207,7 @@ const Component: FC<SingleSelectProps> = memo(
                         onOptionClick={handleOptionClick}
                         maxWidth={maxWidth}
                         includesNestedOptions={includesNestedOptions}
+                        readOnly
```

Comment:
Why did you add `readOnly` ?

Reply:
This was done to not let div get the focus. But since it is now working without it, this can be removed. Thanks